doc/can-bind.md

```diff
@@ -200,7 +200,7 @@ const binding = new Bind({
   parent: parent,
   setChild: function(newValue) {
     const objectValue = deparam(newValue);
-    map.set(objectValue);
+    map.update(objectValue);
```

Comment:
Here’s what this section looks like now:
<img width="902" alt="screen shot 2018-06-28 at 5 38 38 pm" src="https://user-images.githubusercontent.com/10070176/42067269-2b3b72a4-7afa-11e8-8265-4f79ef6e2121.png">